packages/react-native/Libraries/Share/Share.d.ts

```diff
@@ -27,17 +27,11 @@ export type ShareOptions = {
   anchor?: number | undefined;
 };
 
-export type ShareSharedAction = {
-  action: 'sharedAction';
+export type ShareAction = {
```

Comment:
After this change, it is valid to check for `activityType`, but it is always null. So, we should either:
1. Keep this as-is, to require discriminating against the `action` field before checking `activityType`.
1. Add `null` to the list of possible types of `activityType` (unlike Flow, TS optionals may not be null).

Reply:
Hey Nick! Thanks for the review. You're right, I'm going to add null to the activityType typing, It aligns better with other types used in the codebase (this "... | null | undefined" approach), and also avoids making the type a little more complex.